src/server/scripts/Kalimdor/zone_moonglade.cpp

```diff
@@ -300,6 +300,9 @@ class npc_clintar_spirit : public CreatureScript
         {
             Initialize();
             PlayerGUID.Clear();
+            me->SetDisplayId(me->GetCreatureTemplate()->Modelid1);
+            me->RemoveFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NOT_SELECTABLE);
+            me->RemoveFlag(UNIT_FIELD_FLAGS, CREATURE_FLAG_EXTRA_TRIGGER);
```

Comment:
Yes the npc have this flags on sniffs, they are removed when the script starts, they are used to make it invisible to the player if the npc is not active.